android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/BrowserState.kt

```diff
@@ -50,4 +51,5 @@ data class BrowserState(
     val restoreComplete: Boolean = false,
     val locale: Locale? = null,
     val showExtensionProcessDisabledPopup: Boolean = false,
+    val shoppingSheetExpanded: Boolean = false,
```

Comment:
I don't think this should be in the `BrowserState` since the state here shouldn't need to know anything about the application UI state. The Browser state should be limited to properties related to the browser features (used loosely here).

Reply:
Fair enough. Do you have an alternative location in mind? AppState seemed too top-level, but seems more appropriate than `BrowserState` now.

Reply:
I think you're looking at `AppStore`. Here's a prior write-up of the rationale of the `AppStore` https://github.com/mozilla-mobile/fenix/issues/20704#issuecomment-956522471

Reply:
Updated